beam-common/src/main/java/org/gbif/pipelines/config/InterpretationType.java

```diff
@@ -0,0 +1,17 @@
+package org.gbif.pipelines.config;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Defines the types of the interpretations.
+ */
+public enum InterpretationType {
+
+  ALL, LOCATION, TEMPORAL, TAXONOMY, COMMON;
+
+  public static final List<InterpretationType> ALL_INTERPRETATIONS =
```

Comment:
use public static final EnumSet<InterpretationType> ALL_INTERPRETATIONS =
    EnumSet.of(LOCATION, TEMPORAL, TAXONOMY, COMMON); 
instead, why we need ALL_INTERPRETATION if we have an ALL literal

Reply:
changed